reporters/reporter-default/src/main/java/org/radargun/reporting/csv/CsvReporter.java

```diff
@@ -287,4 +325,95 @@ private double round(double number, int places) {
       return bigDecimal.doubleValue();
    }
 
+   /**
+    * Comparator for sorting rows based on column value
+    * 
+    * @author zhostasa
```

Comment:
Yep, the PR was never integrated - though my opinion is that the tags get obsolete too often and VCS is much better in preserving the authorship information - that's why I was removing my own tags only in the end.
Whoever wants to put their name there, can do (I would only ask for full name, not just username).